aws-lc-rs/src/ec.rs

```diff
@@ -6,7 +6,7 @@
 use crate::error::{KeyRejected, Unspecified};
 use core::fmt;
 
-use crate::ptr::{ConstPointer, DetachableLcPtr, LcPtr};
+use crate::ptr::{ConstPointer, DetachableLcPtr, LcPtr, Pointer};
```

Comment:
Nit: Is the `Pointer` import needed?

Reply:
Yeah, it's needed for the use of `as_const_ptr` which is [defined on that trait](https://github.com/aws/aws-lc-rs/blob/e8632a4f80ad059e644b23e6e80c7988a4b7646a/aws-lc-rs/src/ptr.rs#L185).